dependencies/README.md

```diff
@@ -0,0 +1,62 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that inspects classpaths for [static linkage check](
+../library-best-practices/glossary.md#static-linkage-check).
+The tool identifies static linkage errors that appear in Java projects at runtime.
+It scans the class files in a classpath provided as input for references
+to other classes. Then it verifies compatibility of the methods and fields
+in the referenced classes, with regard to their type signatures and accessors.
```

Comment:
I feel like this last sentence is repeating information from "static linkage check." I think it might be more relevant to mention entry point classes and reachability here (since "static linkage check" doesn't imply those).